ee/app/livechat-enterprise/server/hooks/afterForwardChatToAgent.ts

```diff
@@ -0,0 +1,12 @@
+import { callbacks } from '../../../../../app/callbacks/server';
+import LivechatRooms from '../../../../../app/models/server/models/LivechatRooms';
+
+callbacks.add('livechat.afterForwardChatToAgent', (options: { rid?: string } = {}) => {
+	const { rid } = options;
+
+	const room = LivechatRooms.findOneById(rid);
+	if (!room) {
+		return;
```

Comment:
You can't just return anything, callbacks are called in sequence and the return of one callback is pass to the next one, so you always need to return the same inputs you get as parameters.

Reply:
Thanks for this info, wasn't aware of this :sweat_smile: